Default New Promotion Screen/Src/NewPromotionScreenbyDefault/Classes/NPSBDP_UIArmory_PromotionHero.uc

```diff
@@ -680,24 +684,47 @@ function PreviewAbility(int Rank, int Branch)
 			{
 				// Look back to the previous rank and check to see if that ability is a prereq for this one
 				// If so, display a message warning the player that there is a prereq
+				// Start Issue #128
 				foreach AbilityTemplate.PrerequisiteAbilities(PrereqAbilityName)
 				{
-					PreviousAbilityTemplate = AbilityTemplateManager.FindAbilityTemplate(PrereqAbilityName);
-					if (PreviousAbilityTemplate != none && !Unit.HasSoldierAbility(PrereqAbilityName))
+					if (InStr(PrereqAbilityName, "NOT_") == 0)
```

Comment:
```suggestion
					if (InStr(PrereqAbilityName, "NOT_", , true) == 0)
```
Plain `InStr` on coerced names is bad because names are supposed to be case insensitive. `true` as the fourth function argument fixes this by making `InStr` case insensitive. I realize the Highlander also doesn't do this; can you tack on the change to https://github.com/X2CommunityCore/X2WOTCCommunityHighlander/pull/601 ?

Reply:
@robojumper Is the `Repl()` below also case sensitive? Perhaps it would be better to use `Mid()` to remove the first 4 characters, since we've already checked it starts with "NOT_". I do wonder whether @Musashi1584 wanted to make this prefix case sensitive though.